### ISSUE TITLE
Commands frequently require user interaction

### ISSUE THREAD

**Issue by anaisbetts** [WillFantom/OverCLIrr#5]
Commands requiring user interaction means that they cannot be used for automation, one of the biggest reasons to use a CLI - it would be super useful if commands (esp. search) had non-interactive options

**Comment by WillFantom** [comment]
Yeah, kinda went through a phase where I was super into TUIs... I do plan to blitz though a few improvements to this (and the api client package [`goverseerr`](https://github.com/WillFantom/goverseerr/tree/main)) at some point in the very near future, and will definitely add some functions for this.

Will probably take the form of a `--json` flag so a bash script with a little `jq` magic could help automate a few things.

**Comment by anaisbetts** [comment]
Here's a scenario if you want one to design around - because I'm _ancient_, I have a bunch of extremely old CD1/CD2 releases at 480p that I want to write some scripting to re-download

So, I need commands to:

1. Search media by title
2. Delete the release in Overseer (I didn't realize this, but if Overseer thinks a title is in your library, it doesn't let you download it)
3. Queue the request if the title is found
